AndroidSDKCore/src/main/java/com/leanplum/internal/Constants.java

```diff
@@ -157,6 +157,7 @@ public static boolean isNoop() {
     public static final String INBOX_MESSAGE_ID = "newsfeedMessageId";
     public static final String INBOX_MESSAGES = "newsfeedMessages";
     public static final String PARAMS = "params";
+    public static final String REQ_ID= "reqId";
```

Comment:
I don't think it's "consistency" to keep putting constants here. I think you can put it in a better scoped place.

Reply:
Can't think of a better place for this particular one. Ideas?

Reply:
Request.java seems perfect to me. Also, we are using `"reqId"` for efficiency in our requests. This does not mean we want to use abbreviated variable names, which is against LP standards. We can call this `REQUEST_ID_KEY`. Even though its content is `"reqId"`, the name is still fully descriptive.